planner/core/testdata/ordered_result_mode_suite_out.json

```diff
@@ -417,12 +417,12 @@
       },
       {
         "Plan": [
-          "Sort_9 6400.00 root  Column#5, Column#6, Column#7",
-          "└─Selection_11 6400.00 root  lt(Column#6, 20)",
-          "  └─HashAgg_16 8000.00 root  group by:test.t1.d, funcs:min(Column#11)->Column#5, funcs:max(Column#12)->Column#6, funcs:sum(Column#13)->Column#7",
-          "    └─TableReader_17 8000.00 root  data:HashAgg_12",
-          "      └─HashAgg_12 8000.00 cop[tikv]  group by:test.t1.d, funcs:min(test.t1.a)->Column#11, funcs:max(test.t1.b)->Column#12, funcs:sum(test.t1.c)->Column#13",
-          "        └─TableFullScan_15 10000.00 cop[tikv] table:t1 keep order:false, stats:pseudo"
+          "Selection_8 6400.00 root  lt(Column#6, 20)",
```

Comment:
Why the selection_8 can't push down to sort?

Reply:
It's a tradeoff. For the correct result. Actually, it can be pushed down to sort.